scripts/feature_tests/upgrade/controlplane_upgrade/3cp_1w_k8sVer_bootDiskImage_scaleInWorker_upgrade.sh

```diff
@@ -32,10 +32,11 @@ manage_node_taints "${CLUSTER_APIENDPOINT_IP}"
 
 scale_workers_to 0
 worker_has_correct_replicas 0
+expected_free_nodes 1
```

Comment:
Is this used anywhere?

Reply:
yes this checks and waits that one node is in 'ready' state before continuing

Reply:
oops, I missed that it is a function call, thanks